api/types/app.go

```diff
@@ -456,6 +458,23 @@ func (a *AppV3) checkTCPPorts() error {
 	return nil
 }
 
+// GetIdentityCenter returns the Identity Center information for the app, if any.
+// May be nil.
+func (a *AppV3) GetIdentityCenter() *AppIdentityCenter {
+	return a.Spec.IdentityCenter
+}
+
+// GetDisplayName fetches a human-readable display name for the App.
+func (a *AppV3) GetDisplayName() string {
+	// Only Identity Center apps have a display name at this point. Returning
```

Comment:
Technically Okta apps also have a different display name because their actual name is an internal Okta ID. What do we use for them?